### ISSUE TITLE
evaluate problem on DAVIS2017

### ISSUE THREAD

**Issue by limoran233** [andr345/frtm-vos#15]
Hello, When I run code："python evaluate.py --model rn18_all.pth --fast --dset dv2017val", it always appears 
"Computing J-scores
1/30: bike-packing: 2 objects
Traceback (most recent call last):
  File "evaluate.py", line 164, in <module>
    evaluate_dataset(dset, out_path, measure='J')
  File "/new/personal/limoran/frtm-vos/lib/evaluation.py", line 66, in evaluate_dataset
    _print("joint {obj}: acc {score:.3f} \u250a{apf}\u250a".format(obj=obj_id, score=s, apf=text_bargraph(score)))
  File "/new/personal/limoran/frtm-vos/lib/evaluation.py", line 20, in _print
    print(msg)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 19-89: ordinal not in range(128)"
I don't know why and how to modify the code, could you help me ?

**Comment by andr345** [comment]
The cause is that the string contains unicode (utf-8) characters, but print() tries to encode them as ascii.
Are you possibly using python 2.7? You need python 3 for this.
If you are using python3, try forcing python to use unicode, like this:

PYTHONIOENCODING=utf-8 python evaluate.py --model rn18_all.pth --fast --dset dv2017val

**Comment by limoran233** [comment]
Thank you very much for your reply, the problem has been solved by this method

**Comment by limoran233** [comment]
hello! I used the code you gave me, and the result changed a little, but it still didn't work. The result is as follows:
![image](https://user-images.githubusercontent.com/59075406/87854580-7de04e80-c945-11ea-84eb-4a66629ae32c.png)
Moreover, when I set the 65th, 66th, 79th, and 80th in evaluation.py as comments using #, the code "python evaluate.py --model rn18_all.pth --fast --dset dv2017val" can run normally, but acc can't display.

**Comment by andr345** [comment]
I think the easiest way to work around this for you, is to remove the text-bargraph plots so that you can at least print the numbers.

Remove the bargraph by rewriting the print statements (on lines 66 and 79 I think) to something like this:
_print("joint {obj}: acc {score:.3f}".format(obj=obj_id, score=s))

Hopefully, there are no utf8 characters printed now, and you should get some output.

**Comment by limoran233** [comment]
Thank you very much, the problem has been solved.
There is another question. When I used the weights you gave to measure the speed on the 2080TI,
![image](https://user-images.githubusercontent.com/59075406/88612542-071e2080-d0be-11ea-9f96-60870c8ab1d2.png)
the DV16 speed was okay, but the DV17 speed is very different. Should this be the case? When measuring speed with fast and normal models, besides running instructions, are there any other codes that need to be changed?
